assets/js/blocks/filter-wrapper/index.tsx

```diff
@@ -20,6 +21,97 @@ import { isFeaturePluginBuild } from '@woocommerce/block-settings';
 import edit from './edit';
 import metadata from './block.json';
 
+const filterBlocksWidgets = [
+	{
+		widgetId: 'woocommerce_layered_nav_filters',
+		name: 'active-filters',
+		heading: __( 'Active filters', 'woo-gutenberg-products-block' ),
+	},
+	{
+		widgetId: 'woocommerce_price_filter',
+		name: 'price-filter',
+		heading: __( 'Filter by price', 'woo-gutenberg-products-block' ),
+	},
+	{
+		widgetId: 'woocommerce_layered_nav',
+		name: 'attribute-filter',
+		heading: __( 'Filter by attribute', 'woo-gutenberg-products-block' ),
+	},
+	{
+		widgetId: 'woocommerce_rating_filter',
+		name: 'rating-filter',
+		heading: __( 'Filter by rating', 'woo-gutenberg-products-block' ),
+	},
+];
+
+const getTransformAttributes = ( instance, filterType: string ) => {
+	switch ( filterType ) {
+		case 'attribute-filter':
+			return {
+				attributeId: 0,
+				showCounts: true,
+				queryType: instance?.raw?.query_type || 'or',
+				heading: '',
+				displayStyle: instance?.raw?.display_type || 'list',
+				showFilterButton: false,
+				selectType: instance?.raw?.select_type || 'multiple',
+				isPreview: false,
+			};
+		case 'active-filters':
+			return {
+				displayStyle: 'list',
+				heading: '',
+			};
+		case 'price-filter':
+			return {
+				heading: '',
+				showInputFields: false,
+				showFilterButton: true,
+				inlineInput: false,
+			};
+		default:
+			return {};
+	}
+};
+
+const isFilterWidget = ( widgetId: string ) =>
+	filterBlocksWidgets
+		.map( ( item ) => item.widgetId )
+		.filter( Boolean )
+		.includes( widgetId );
```

Comment:
I think this can be simplified to:
```suggestion
	filterBlocksWidgets
		.some( ( item ) => item.widgetId === widgetId );
```